__init__.py

```
@@ -279,16 +324,68 @@
 
+    @staticmethod
+    def close_window(window_id, wmctrl="wmctrl"):
+        try:
+            # Get the list of windows with wmctrl
+            result = subprocess.run([wmctrl, '-ic', window_id])
+            # windows are returned sorted by order of creation, but we dont have that timestamp
+            # TODO - is this true or just coincidence in my tests? i don't think it is ensured
+            if result.returncode != 0:
+                LOG.error("'wmctrl' command failed.")
+            return True
+        except:
```

Ruff

Do not use bare `except`

(E722)